### ISSUE TITLE
Contribute this package to JuliaSparse

### ISSUE THREAD

**Issue by ViralBShah** [JuliaSparse/HarwellRutherfordBoeing.jl#3]
@dpo I wonder what you think about moving this package to the JuliaSparse organization. We are doing this in order to collect various sparse matrix related codes in one umbrella organization. Of course, completely ok, if you prefer to keep things as they are.

It would also be great if the package could be licensed MIT/BSD.

**Comment by dpo** [comment]
Sure that sounds good! I'll add the Rutherford Boeing reader and update the license this weekend.

**Comment by ViralBShah** [comment]
Thanks!

**Comment by dpo** [comment]
Should be good to go. I renamed the module `HarwellRutherfordBoeing`. I added a preliminary Rutherford-Boeing reader. I haven't tested it on the entire collection, but it's been working diligently so far. The Harwell-Boeing reader has been tested on the entire collection.

Once the transition is done, I'll set up TravisCI and Appveyor builds. My experience is that Appveyor doesn't like when repos move or change name.
